.github/workflows/continuous-integration.yml

```diff
@@ -30,8 +30,9 @@ jobs:
         uses: actions/cache@v1.0.0
         with:
           path: ~/.composer/cache
-          key: php7.2-composer-locked-${{ github.sha }}
+          key: php7.2-composer-locked-${{ hashFiles('**/composer.lock') }}
           restore-keys: |
+            php7.2-composer-locked-${{ hashFiles('**/composer.lock') }}
```

Comment:
what is this for? isn't it redundant?
The `key` and `restore-keys` are already merged for restoring:
https://github.com/actions/cache/blob/bc821d0c12b430b965fdc132e1a32a346ce876d4/src/restore.ts#L26

Reply:
Yes, looking at the code it's obviously redundant.